frontend/src/App.tsx

```diff
@@ -303,38 +302,64 @@ services:
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="upsun merge staging">
-                        <code className="px-4">upsun merge staging</code>
-                      </CopyButton>
-                    </p>
-                    <p className="mb-2 mt-4">
-                      You will once again need to set the resources for the production environment and its new service, so rerun the command below
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="upsun resources:set --size redis_persistent:0.5 --disk redis_persistent:512 -e main">
-                        <code className="px-4">upsun resources:set --size redis_persistent:0.5 --disk redis_persistent:512 -e main</code>
-                      </CopyButton>
-                    </p>
-                    <p className="mb-2 mt-4">
-                      When the activity has completed, you can revisit the production environment state by running the command below:
-                    </p>
-                    <p className="mb-2 mt-4">
-                      <CopyButton className="hidden sm:inline-block w" copyText="upsun url --primary -e main">
-                        <code className="px-4">upsun url --primary -e main</code>
-                      </CopyButton>
-                    </p>
+
+                    {environment?.toLocaleLowerCase() === "production"
+                        ? "Use your preview environments to stage any future updates."
+                        : "Use this or other preview environments to stage any future updates."
+                      }
+                      
+                    </p>
+                    <h4 className="mt-5 text-lg font-semibold">Next Step</h4>
+                    <ol className="list-decimal list-outside ml-4 mt-2">
+                      <li>
+                        <p className="mb-2">
+                          <span>Deploy staging changes to production</span>
+                          <CopyButton className="pl-1 inline-block w-full" copyText="upsun merge staging">
+                            <p className="mb-2 mt-2 code-block">
+                              <CodeBlock
+                                text="upsun merge staging"
+                                showLineNumbers={false}
+                                theme={dracula}
+                              />
+                            </p>
+                          </CopyButton>
+                        </p>
+                      </li>
+                      <li>
+                        <p className="mb-2 mt-2">
+                          <span>Allocate resources to Redis in production</span>
+                          <CopyButton className="pl-1 inline-block w-full" copyText={`upsun resources:set\n\t--size redis_persistent:0.5\n\t--disk redis_persistent:512\n\t -e main`}>
```

Comment:
copying this block and pasting verbatim fails. If we're expecting the user to copy & paste the commands then we need to add the line return escapes:
```shell
upsun resources:set \
	--size redis_persistent:0.5 \
	--disk redis_persistent:512 \
	 -e main
```